src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/judgeapproval/JudgeApprovalResolverTest.java

```diff
@@ -69,11 +69,14 @@ class JudgeApprovalResolverTest {
     @ParameterizedTest
     @MethodSource("provideShouldInvokeProcessHearingInstructionData")
     void shouldInvokeProcessHearingInstruction(DraftOrdersWrapper draftOrdersWrapper, int expectHearingInvocationCount) {
+
+        JudgeApproval judgeApproval = JudgeApproval.builder().judgeDecision(READY_TO_BE_SEALED).build();
+
         // Execute the method being tested
         judgeApprovalResolver.populateJudgeDecision(FinremCaseDetails.builder().build(),
             draftOrdersWrapper,
             CaseDocument.builder().build(),
-            JudgeApproval.builder().judgeDecision(READY_TO_BE_SEALED).build(),
+            judgeApproval,
```

Comment:
It seems you didn't make any change on this bit.

Reply:
[4000528](https://github.com/hmcts/finrem-case-orchestration-service/pull/2048/commits/4000528df9cfa70aa158b1acff9485de4b5a914e)